processor/signozspanmetricsprocessor/processor.go

```diff
@@ -501,12 +503,14 @@ func (p *processorImp) collectExternalCallMetrics(ilm pmetric.ScopeMetrics) erro
 	mExternalCallSum := ilm.Metrics().AppendEmpty()
 	mExternalCallSum.SetName("signoz_external_call_latency_sum")
 	mExternalCallSum.SetUnit("1")
-	mExternalCallSum.SetEmptySum().SetAggregationTemporality(p.config.GetAggregationTemporality())
+	mExternalCallSum.SetEmptySum().SetIsMonotonic(true)
```

Comment:
This change makes no functional difference. We do this so the `temporality` column is populated with the correct value.